bundles/af-core/src/test/resources/form/image/exporter-image-parsedSrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "image-7cfd7f1fe4",
+  "fieldType": "image",
+  "name": "abcd",
+  "visible": false,
+  "altText": "abc",
+  "value": "/content/image-customized.img.png",
+  "source": "/content/image-customized.img.png",
```

Comment:
The schema validation test case succeeded because it currently only verifies required properties for specific field types. However, ideally, we should also review and possibly update the specification for source/value property name. Can you start a conversation on our internal Slack channel to finalize the property name? In my opinion, the source should be a custom property that points to the DAM image, given that this detail is specific to AEM. The specification should mandate only a single property for an image, which is its URL (whether the property name should be value or source, we need to decide the same, but only one property should be enforce by spec)